runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -992,6 +993,34 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadSpaceToBatchND(const Operator
   subg.addOperation(std::move(new_op));
 }
 
+template <typename LoaderDomain, typename SpecificLoader>
+void BaseLoader<LoaderDomain, SpecificLoader>::loadBatchMatMul(const Operator *op, ir::Graph &subg)
+{
+  ir::OperandIndexSequence inputs;
+  ir::OperandIndexSequence outputs;
+
+  loadOperationIO(op, inputs, outputs);
+  ir::operation::BatchMatMul::Param param;
+
+  if (op->custom_options() == nullptr)
+  {
+    param.adj_x = false;
+    param.adj_y = false;
+  }
+  else
+  {
+    size_t custom_op_data_size = op->custom_options()->size();
+    auto custom_op_data = op->custom_options()->Data();
+    auto data_root = flexbuffers::GetRoot(custom_op_data, custom_op_data_size);
+    auto attr_map = data_root.AsMap();
+    param.adj_x = attr_map["adj_x"].AsBool();
+    param.adj_y = attr_map["adj_y"].AsBool();
```

Comment:
@hseok-oh When `adj_x` or `adj_y` don't exist in flexbuffer, it does not throw error, but it returns empty `Reference`, which will return `false` fo `AsBool()`. ( I've checked by reading flatbuffer source.). It is same to [default value (=`false`)](https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/batch-mat-mul#summary) of `adj_x` and `adj_y`. Thus it is okay. Am I right?

Reply:
Exactly.